caos.ansible_roles/roles/cleanup/tasks/files-Win32NT.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: remove infra-agent files and directories
```

Comment:
Is this tied to the infra-agent? Should we change the name?

Reply:
Nice 👁️ Done in d832c855867ff293db534e296aed06c9009ef371